Content guidelines.md

```diff
@@ -0,0 +1,65 @@
+# Content guidelines
+
+We know that your platform has its own voice and tone, and it’s ok to preserve them in the channel. However, channels are part of Shopify and merchants don’t see them as a separate experience.
+
+When you’re creating content for a sales channel, use Shopify’s [voice and tone guide](https://polaris.shopify.com/content/voice-and-tone) to validate your decisions.
+
+> Note
+>
+> > When building the channel, use the names of features you have on your platform without altering, so that merchants can recognize them when they leave Shopify admin and use your platform. 
```

Comment:
```suggestion
> > When building the channel, don't change the names of features on your platform, so that merchants can recognize them when they leave the Shopify admin and use your platform. 
```